include/zephyr/bluetooth/audio/bap.h

```diff
@@ -41,9 +41,73 @@
 extern "C" {
 #endif
 
+#if defined(CONFIG_BT_BAP_BASS_MAX_SUBGROUPS)
+#define BT_BAP_BASS_MAX_SUBGROUPS CONFIG_BT_BAP_BASS_MAX_SUBGROUPS
+#else
+#define BT_BAP_BASS_MAX_SUBGROUPS 0
+#endif /* CONFIG_BT_BAP_BASS_MAX_SUBGROUPS*/
+
 /** An invalid Broadcast ID */
 #define BT_BAP_INVALID_BROADCAST_ID 0xFFFFFFFFU
 
+/**
+ * @name Suggested connection and advertising intervals suited for coexistence with audio
+ *
+ *
+ * Suggest use is to use this as both the minimum and maximum interval for best results.
+ * The intervals are suited for both unicast and broadcast.
+ * @{
+ */
+/**
```

Comment:
Some of these are not used by any sample or test. Should we remove one of the "FAST" ones, or keep the unused ones?

Reply:
Since these are suggestions to the application developer of good intervals to use we can keep the FAST ones, even when they are not used in a sample IMO